src/config.rs

```diff
@@ -92,6 +93,7 @@ macro_rules! build_cf_opt {
     ($opt:ident) => {{
         let mut block_base_opts = BlockBasedOptions::new();
         block_base_opts.set_block_size($opt.block_size.0 as usize);
+        block_base_opts.set_no_block_cache($opt.disable_block_cache);
```

Comment:
What will happen if `no_block_cache = true` and `block_size` is not 0?

Reply:
RocksDB will reset the block cache if `no_block_cache = true`.